cadence/server/public/js/aria.js

```diff
@@ -1,190 +1,146 @@
-var streamSrcURL = "";
+const SONG_ID = '#song'
+const ARTIST_ID = '#artist'
+const ARTWORK_ID = '#artwork'
+const STATUS_ID = '#status'
+const SEARCH_INPUT_ID = '#searchInput'
+const LISTENERS_ID = '#listeners'
+const RELEASE_ID = 'release'
+const STREAM_ID = 'stream'
+const REQUEST_STATUS_ID = 'requestStatus'
 
-$(document).ready(function () {
-	getListenURL();
-	getNowPlayingMetadata();
-	getNowPlayingAlbumArt();
-	getVersion();
-	connectRadioData();
-	postSearch();
-	postRequestID();
-});
+let streamSrcURL = ''
 
-function getVersion() {
-	$.ajax({
-		type: "GET",
-		url: "/api/version",
-		dataType: "json",
-		success: function (data) {
-			document.getElementById("release").innerHTML = data.Version;
-		},
-		error: function () {
-			document.getElementById("release").innerHTML = "(N/A)";
-		},
-	});
+$(document).ready(() => {
+   getListenURL()
+   getNowPlayingMetadata()
+   getNowPlayingAlbumArt()
+   getVersion()
+   connectRadioData()
+   postSearch()
+   postRequestID()
+})
+
+let getVersion = () => {
```

Comment:
Why is this a `let`? I don't think we should be reassigning these functions.

(etc for the others)